### ISSUE TITLE
Save warn apears when hitting Save.  Also, does not always save

### ISSUE THREAD

**Issue by dougcassidy** [reduxframework/redux-framework#1767]
Redux Framework v3.3.8.6 Version 4.0
WP version 4.0
dev_mode = either
Redux running within my plugin
no js errors
1. type in box
2. Hit save button
   Expected: Settings are saved.
   Actual: Save warning appears, then settings are saved.  

Sometimes, Save warning appears and nothing else happens, need to hit Save, again, but this is on Coowon (derived from chrome) FF and C dont seem to have that problem.  But FF and C do show the Save warning.

Basically, I'm saying that if I hit the Save button, I shouldnt see the Save Warning.
